packages/terra-form-select/tests/jest/Tag.test.jsx

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
 import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
-
+import { IntlProvider } from 'react-intl';
 import Tag from '../../src/shared/_Tag';
 
 describe('Tag', () => {
   it('should render a default Tag', () => {
-    const wrapper = enzyme.shallow(<Tag value="value" onDeselect={() => {}}>Content</Tag>);
+    const wrapper = enzyme.shallow(
```

Comment:
Use `shallowWithIntl` no need to wra with IntlProvider. See tests in other components for reference